docs/UserGuide.md

```diff
@@ -282,7 +305,7 @@ Examples:
 - `find Betsy` followed by `delete 1` deletes the 1st person in the Student's Details panel.
 
 ```yaml
-❗ Caution: Deleting a student is irreversible! Please input the correct index number(s).
+❗ Caution: Deleting a student by mistake can be reversed by "undo" command.
```

Comment:
not sure if can add a hyperlink here inside yaml. if can, would be good :)

Reply:
Have researched on this topic and cannot seem to find a quick fix :(